--- origsrc/cooka-0.3/Makefile.cooka	2007-02-21 22:50:29.000000000 -0600
+++ src/cooka-0.3/Makefile.cooka	2007-02-22 22:41:08.234375000 -0600
@@ -15,8 +15,8 @@
 RESOURCES_BUILD = $(foreach res,$(RESOURCES), $(addprefix -resource:,$(res)),$(notdir $(res)))
 
 PKG_REFERENCES = \
-gtk-sharp \
-glade-sharp 
+gtk-sharp-2.0 \
+glade-sharp-2.0
 
 PKG_REFERENCES_BUILD = $(addprefix -pkg:, $(PKG_REFERENCES))
 
